### PR TITLE
Hosted website

### PR DESCRIPTION
**Pull request by cdelapena**
PR for files specific to our hosted website: [https://cs467-jhaiwt-431408.wl.r.appspot.com/](https://cs467-jhaiwt-431408.wl.r.appspot.com/)

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import Home from "./Pages/Home/Home";
 import ResultsPage from "./Pages/ResultsPage/ResultsPage";
 import BrowsePage from "./Pages/BrowsePage/BrowsePage";
 import { getDesignTokens } from "./shared/colorTheme";
-import Navbar from "./components/Navbar";
+import Navbar from "./components/navbar";
```
Comment (bryanw121) [inline frontend/src/App.tsx:18]: this should technically stay capitalized. You can change it from lowercase to capital in the file name itself

File: backend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
```
Comment (bryanw121) [inline backend/Dockerfile:1]: are we actually running this docker container? just curious bc it looks like flask run is the command, but we end up using gunicorn

**Review by bryanw121 — COMMENTED** [review]
looks good! just a couple comments and then I'll approve

File: frontend/src/App.tsx (hunk 1 above)
Comment (cdelapena) [reply frontend/src/App.tsx]: sounds good, done

File: backend/Dockerfile (hunk 2 above)
Comment (cdelapena) [reply backend/Dockerfile]: No, the Dockerfiles are remnants that were replaced by the app.yaml files. Dockerfiles are removed.

**Review by bryanw121 — APPROVED** [review]
looks good to me!

**Pull request merged**